scripts/validate-sysadmin-harness.ps1

```
@@ -0,0 +1,132 @@
+[CmdletBinding()]
+param()
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$repoRoot = (Get-Location).Path
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'repoRoot' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)